dwi_ml/data/hdf5/hdf5_creation.py

```diff
@@ -641,11 +675,10 @@ def _process_one_streamline_group(
             The Euclidean length of each streamline
         """
         tractograms = self.groups_config[group]['files']
-
         if self.step_size and self.compress:
             raise ValueError(
```

Comment:
nb_points is not checked. But now, you check that in the `__init__`, so we can remove the check here.